browser-extension/addTournesolRecommendations.js

```diff
@@ -78,6 +86,45 @@ const getTournesolComponent = (data) => {
     tournesol_link.append('learn more');
     inline_div.append(tournesol_link);
 
+    // Refresh button
+    refresh_button = document.createElement('button');
+    refresh_button.setAttribute('id', 'tournesol_refresh_button');
+    fetch(chrome.runtime.getURL('images/sync-alt.svg'))
+      .then(r => r.text())
+      .then(svg => refresh_button.innerHTML = svg);
+
+    refresh_button.className = 'tournesol_simple_button';
+    refresh_button.onclick = () => {
+      refresh_button.disabled = true;
+      loadRecommandations(loadVideos = true, loadAdditionalVideos = isExpanded);
+    };
+    inline_div.append(refresh_button);
+    // Expand button
+    expand_button = document.createElement('button');
+    expand_button.setAttribute('id', 'tournesol_expand_button');
+    // A new button is created on each video loading, the image must be loaded accordingly
+    if(isExpanded){
+      fetch(chrome.runtime.getURL('images/chevron-down.svg'))
```

Comment:
The `chevron-down.svg` and `chevron-up.svg` could be swapped.

Currently click on the up chevron will expand the panel downward, and clicking on the up chevron will expand the panel upward. We found with @amatissart that's it more natural to have the chevron pointing in the same direction where the panel will expand.

Reply:
To keep the extension user-friendly, we could in a future task, decide where we want to put the refresh and the expand buttons.

With @amatissart we had the idea of moving the expand button below the recommendation panel. Right it's just an idea, but it raises the question where do we want to place the action buttons.

Reply:
Yes I think the expand/reduce button would be better positioned below the panel. 

We could handle this in a separate PR though, with a cleaner layout for existing buttons. The current style with absolute positioning and floating elements is a bit difficult to work with.

Reply:
Agreed!